ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -186,13 +189,18 @@ class IdpLoginScreen extends React.Component<Props, State> {
                 : "authentication.errors.login.title"
             )}
           </Text>
-          <Text style={styles.errorBody}>
-            {I18n.t(
-              errorType === "LOADING_ERROR"
-                ? "authentication.errors.network.body"
-                : "authentication.errors.login.body"
-            )}
-          </Text>
+
+          {errorType === "LOGIN_ERROR" && (
+            <Text style={styles.errorBody}>
+              {this.state.errorCode
```

Comment:
this issue is not solved yet 

> you have no guarantee the incoming error code is mapped into translation keys
> I suggest mapping the handled errors and if an error code is not mapped, you should show a generic error message

what if you have a URL like `.../error.html?errorCode=27`? This will generate an error because you don't have `authentication.errors.spid.error_27` translation key

Reply:
True! I try to solve in this way: 36b2cb3